packages/kbn-cell-actions/src/components/inline_actions.tsx

```diff
@@ -39,7 +44,12 @@ export const InlineActions: React.FC<InlineActionsProps> = ({
   );
 
   return (
-    <span data-test-subj="inlineActions">
+    <EuiFlexGroup
+      alignItems="flexStart"
+      gutterSize="none"
+      data-test-subj="inlineActions"
+      className={`inlineActions ${isPopoverOpen ? 'inlineActions-popoverOpen' : ''}`}
+    >
```

Comment:
Shouldn't we render at least one `EuiFlexItem` inside the `EuiFlexGroup`? I don't see any